pkg/storage/segments/store.go

```diff
@@ -54,17 +54,34 @@ type Store interface {
 	List(ctx context.Context, prefix, startAfter, endBefore storj.Path, recursive bool, limit int, metaFlags uint32) (items []ListItem, more bool, err error)
 }
 
+type nodeStats struct {
+	Uptime       int
+	UptimeCount  int
+	AuditSuccess int
+	AuditCount   int
+	Excluded     storj.NodeIDList
```

Comment:
`Excluded` is not part of node stats. It also doesn't look like you are using it anywhere.

I also think you should consider using `pb.NodeStats` rather than defining your own struct: https://github.com/storj/storj/blob/master/pkg/pb/node.proto#L52

`AuditSuccess` and `Uptime` should both be `float64`s if you are planning on keeping this struct definition